scripts/src/bundle/AdapterBundle.ts

```diff
@@ -0,0 +1,34 @@
+import path from "path";
+import chalk from "chalk";
+
+import { BundleInfo, PlatformType } from "./BundleInfo.js";
+import { rootDir } from "../cli.js";
+import { getPlatformsFromPath, normalizePath } from "../utils/Utils.js";
+
+export function getMinigameAdapterBundle(bundleName, platformType: PlatformType): BundleInfo[] {
```

Comment:
The function name is somewhat contradictory, perhaps naming it 'getMiniAdapterBundle' would be more appropriate

Reply:
Has changed to `getPolyfillBundle`.